projects/components/src/table/cells/csv-generators/table-cell-boolean-csv-generator.ts

```diff
@@ -0,0 +1,13 @@
+import { TableCellCsvGeneratorBase } from '../table-cell-csv-generator-base';
+import { Injectable } from '@angular/core';
+import { isUndefined } from 'lodash-es';
+
+@Injectable({ providedIn: 'root' })
+export class TableCellBooleanCsvGenerator implements TableCellCsvGeneratorBase<boolean> {
+  public readonly type = 'CSV_GENERATOR';
```

Comment:
What's the purpose of this value?

Reply:
so that the injection token can correctly pick up all the generators. Without an identifying property like this, it does not work.

Reply:
actually, this may not be needed anymore. let me check

Reply:
removed